static/src/js/util/request/request.ts

```diff
@@ -51,15 +83,15 @@ export default class Request {
    * Filter out any unwanted or non-permitted data
    * @param {Objet} data - An object representing an HTTP request payload
    */
-  filterData(data) {
+  filterData(data: unknown) {
```

Comment:
Create a CMRData type or something like that in sharedTypes.ts to use for these. It can be `any` for now and `// @ts-expect-error` it